lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -54,7 +70,9 @@ def ask_for_repmgrd_configuration
 
     def activate
       say("Configuring Replication Standby Server...")
-      data_dir_empty? &&
+      initialize_postgresql_disk if disk
+      PostgresAdmin.prep_data_directory &&
```

Comment:
agree. how about we just remove the "&&"? if the dir is empty the next action will kick in anyhow, if not it should break.